frontend/packages/core/src/AppProvider/index.tsx

```diff
@@ -214,21 +215,32 @@ const ClutchApp = ({
                             const heading = route.displayName
                               ? `${workflow.displayName}: ${route.displayName}`
                               : workflow.displayName;
+
+                            // We define these props in order to avoid UI changes before refactoring
+                            const workflowLayoutProps: LayoutProps = {
+                              variant: "custom",
+                              hideHeader: true,
+                              heading,
+                              ...route.layoutProps,
+                            };
                             return (
                               <Route
                                 key={workflow.path}
                                 path={`${route.path.replace(/^\/+/, "").replace(/\/+$/, "")}`}
                                 element={
-                                  <AppNotification
-                                    type="layout"
-                                    workflow={workflow?.displayName}
-                                    banners={appConfiguration?.banners}
-                                  >
-                                    {React.cloneElement(<route.component />, {
-                                      ...route.componentProps,
-                                      heading,
-                                    })}
-                                  </AppNotification>
+                                  <WorkflowLayout {...workflowLayoutProps}>
```

Comment:
Sorry, had one thought about this. Let's add an appConfig variable that basically is `useWorkflowSpacing` or something, disabled by default (this way it doesn't negatively affect others) and then we enable it manually.